serializers/neos_serializer.rb

```diff
@@ -0,0 +1,14 @@
+class NeosSerializer
+  def daily(response)
+    neos = []
+    response[:near_earth_objects].first[1].each do |neo|
+      neos << {
+        id: neo[:id],
+        name: neo[:name],
+        relative_velocity: neo[:close_approach_data][0][:relative_velocity][:miles_per_hour],
+        lunar_distance: neo[:close_approach_data][0][:miss_distance][:lunar]
+      }
+    end
+    neos.sort_by! { |neo| neo[:lunar_distance].to_f }[0..3]
```

Comment:
sort by lunar distance (lunar distances in today's example were ~50-100 which is a cool unit of measurement and much easier to read. they are in float with 10 decimal places. we can change these to 2, as well. also returns the closest four.

Reply:
Great stuff here this looks like a really solid template that is gonna work great for the what we need.